src/gui/search/searchwidget.cpp

```diff
@@ -285,7 +285,7 @@ void SearchWidget::giveFocusToSearchInput()
 // Function called when we click on search button
 void SearchWidget::on_searchButton_clicked()
 {
-    if (Utils::Misc::pythonVersion() < 0) {
+    if (Utils::Python::majorVersion() < 0) {
```

Comment:
I can't find where you set negative version number, as such the test might never fail.

Reply:
I don't plan to fix it now, maybe in later PR. I don't mind you push a fix commit onto this branch though.

Reply:
But here you introduce a mistake. The current code initialises mahorVersion to -1 and it is left as such if we fail to find a working python. You change variable type to Utils::Version and initialise it to empty version which is 0.0.0, I believe.

Reply:
Thanks, I'll fix this one.